src/zimdump.cpp

```diff
@@ -302,6 +306,17 @@ inline void write_to_file(const std::string &base, const std::string& path, cons
     close(fd);
 }
 
+void ZimDumper::writeHttpRedirect(const std::string& directory, const std::string& outputPath, const std::string& currentEntryPath, std::string redirectPath)
+{
+    redirectPath = computeRelativePath(currentEntryPath, redirectPath);
+    auto encodedurl = urlEncode(redirectPath, true);
+    std::ostringstream ss;
+
+    ss << "<!DOCTYPE html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\" />";
+    ss << "<meta http-equiv=\"refresh\" content=\"0;url=" + encodedurl + "\" /><head><body></body></html>";
+    auto content = ss.str();
+    write_to_file(directory + SEPARATOR, outputPath, content.c_str(), content.size());
+}
```

Comment:
It would be nice to test this, even if you have not written it (to reach the 80% coverage).
Maybe move the generation of the content in a free function to simplify the test.

Reply:
That was a good advice. The HTML was buggy (not opening `<html>` tag, `<head>` was not properly closed).